addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -50,10 +50,9 @@ export class OutOfFocusService {
                 notificationTitle = author.name;
             }
         }
-        const notificationContent = htmlToTextContentInline(message.body).substring(
-            0,
-            PREVIEW_MSG_MAX_SIZE
-        );
+        const notificationContent = (
+            htmlToTextContentInline(message.body) + message.generateTrackingMessage()
+        ).substring(0, PREVIEW_MSG_MAX_SIZE);
```

Comment:
Should `htmlJoin()` both the body and `message.generateTrackingMessage()` _(please be cautious that `generateTrackingMessage` shows tracking values that may contain user input)_